dimes/dimes.py

```diff
@@ -131,19 +131,39 @@ def finalize_plot(self):
                                         axis.units,
                                     ),
                                     name=time_series.name,
-                                    yaxis=f"y{axis_id}",
+                                    yaxis=f"y{y_axis_id}",
+                                    xaxis=f"x{x_axis_id}",
                                     mode="lines",
                                     visible="legendonly" if not time_series.is_visible else True,
                                     line={
                                         "color": time_series.color,
                                         "dash": time_series.line_type,
                                     },
                                 ),
-                                row=None if number_of_subplots == 1 else subplot_number,
-                                col=None if number_of_subplots == 1 else 1,
                             )
-                        self.fig.layout[f"yaxis{axis_id}"]["title"] = axis.get_axis_label()
+                        is_base_y_axis = subplot_base_y_axis_id != y_axis_id
+                        self.figure.layout[f"yaxis{y_axis_id}"] = {
+                            "title": axis.get_axis_label(),
+                            "domain": subplot_domains[subplot_number - 1]
+                            if subplot_base_y_axis_id == y_axis_id
+                            else None,
+                            "side": y_axis_side,
+                            "anchor": f"x{x_axis_id}",
+                            "overlaying": f"y{subplot_base_y_axis_id}" if is_base_y_axis else None,
+                            "tickmode": "sync" if not is_base_y_axis else None,
```

Comment:
Suggestion to rename `is_base_y_axis` to `is_not_base_y_axis` or something similar.

You may also consider changing line 144 to `is_base_y_axis = subplot_base_y_axis_id == y_axis_id`, and then adjusting lines 152 and 153 to the following:

`"overlaying": f"y{subplot_base_y_axis_id}" if not is_base_y_axis else None,`
`"tickmode": "sync" if is_base_y_axis else None,`